src/DebugBar/DataCollector/PDO/PDOCollector.php

```diff
@@ -65,11 +65,14 @@ public function getSqlQuotationChar()
      * @param TraceablePDO $pdo
      * @param string $name Optional connection name
      */
-    public function addConnection(TraceablePDO $pdo, $name = null)
+    public function addConnection(\PDO $pdo, $name = null)
     {
         if ($name === null) {
             $name = spl_object_hash($pdo);
         }
+        if ($pdo instanceof \PDO) {
+            $pdo = new TraceablePDO($pdo);
```

Comment:
Isn't this always true? Shouldn't it check if it isn't an instanceof TraceablePDO?

Reply:
:man_facepalming:

Reply:
I got caught on this one right now too. But actually `PDOCollector::__construct()` accepts `null` as the `$pdo` param, thus this check is needed so it doesn't try to envelope null instead of a PDO instance in those cases.

Reply:
I'm not sure if it's an unsent or missed notification, or simply the maintainers busy with life and stuff, but anyway I'll ping @barryvdh again. Sorry if that's spammy, but this is fixed :sweat_smile: